docs/en/nebd_en.md

```diff
@@ -0,0 +1,100 @@
+[中文版](../cn/nebd.md)
+
+### Overview
+
+CURVE client serves as the entrance of the services provided by CURVE, providing dynamic link library for QEMU/CURVE-NBD for calling. As a result, restarting QEMU/CURVE-NBD is necessary when CURVE Client needs to be updated.
+
+In order to relief the impact of updating on applications based on CURVE, we decoupled CURVE client and its applications, and imported the hot upgrade module **NEBD** in between.
+
+### NEBD Structure
+
+<p align="center">
+    <img src="../images/nebd-overview.jpg" alt="nebd-overview" width="650" /><br>
+    <font size=3> Figure 1: NEBD structure</font>
+</p>
+
+Figure1 shows the deployment structure of NEBD.
+
+- NEBD Client (**part1** in source code directory): NEBD Client corresponds to applications based on CURVE, including QEMU and CURVE-NBD.  An NEBD client connects to specified NEBD server through Unix Domain Socket.
+
+- NEBD Server(**part2** in source code directory)：NEBD Server is responsible for receiving the requests from part1, then call CURVE client for corresponding operations. An NEBD server can receive requests from different NEBD clients.
+
+
+Also, figure 1 shows that instead of CURVE client, NEBD client is now the component that serves the application above. In this case, the applications will still be influenced when NEBD client is being upgraded. So in our design, we simplified the processing logic of NEBD client as much as possible, which means it will only be responsible for request forwarding and limited retries for requests if needed.
```

Comment:
the processing logic of NEBD -> the business logic of NEBD

Reply:
> the processing logic of NEBD -> the business logic of NEBD

done